modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -0,0 +1,4477 @@
+:page-styles: hide-table-captions
+
+This page provides a complete reference to the Neo4j configuration settings, which can be set in xref::/configuration/file-locations.adoc#file-locations[_neo4j.conf_].
+Refer to xref::/configuration/neo4j-conf.adoc#neo4j-conf[The neo4j.conf file] for details on how to use configuration settings.
+
+Some of the settings are labeled Dynamic, which means that they can be changed at runtime, without restarting the service.
+For more information on how to update dynamic configuration settings, see xref:configuration/dynamic-settings.adoc[Update dynamic settings].
+
+[NOTE]
+====
+Changes to the configuration at runtime are not persisted.
+To avoid losing changes when restarting Neo4j, make sure you update xref:configuration/file-locations.adoc[_neo4j.conf_] as well.
+
+In a clustered environment, `CALL dbms.setConfigValue` affects only the server it is run against, and it is not propagated to other members.
+If you want to change the configuration settings on all cluster members, you have to run the procedure against each of them and update their _neo4j.conf_ file.
+====
+
+* <<Checkpoint settings>>
+* <<Cluster settings>>
+* <<Connection settings>>
+* <<Cypher settings>>
+* <<Database settings>>
+* <<DBMS settings>>
+* <<Import settings>>
+* <<Index settings>>
+* <<Logging settings>>
+* <<Memory settings>>
+* <<Metrics settings>>
+* <<Neo4j Browser and client settings>>
+* <<Kubernetes settings>>
+* <<Security settings>>
+* <<Server directories>>
+* <<Server settings>>
+* <<Transaction settings>>
+* <<Transaction logs settings>>
+
+== Checkpoint settings
+
+Checkpointing is the process of flushing all pending page updates from the page cache to the store files.
+This is done periodically and is used to recover the database in case of a crash.
+The checkpoint settings control the frequency of checkpoints, and the amount of data that is written to disk in each checkpoint.
+//For more information, see <<Checkpointing>>.
+See also, <<Transaction logs settings>>.
+
+[[config_db.checkpoint]]
+
+[[config_db.checkpoint]]
+=== `db.checkpoint`
+
+.db.checkpoint
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|Configures the general policy for when checkpoints should occur. The default policy is the 'periodic' checkpoint policy, as specified by the '<<config_db.checkpoint.interval.tx,db.checkpoint.interval.tx>>' and '<<config_db.checkpoint.interval.time,db.checkpoint.interval.time>>' settings. The Neo4j Enterprise Edition provides two alternative policies: The first is the 'continuous' checkpoint policy, which will ignore those settings and run the checkpoint  process all the time. The second is the 'volumetric' checkpoint policy, which makes a best-effort at checkpointing often enough so that the database doesn't get too far behind on deleting old transaction logs in accordance with the '<<config_db.tx_log.rotation.retention_policy,db.tx_log.rotation.retention_policy>>' setting.
```

Comment:
Some of the descriptions have single quotes instead of backticks, so that setting names are not marked up correctly - I've amended them in this example, but there are a few others (for example, `dbms.cluster.minimum_initial_system_primaries_count`) I know these are matched to the content in the product - are you planning an editorial review of the descriptions, as I think paragraph breaks would be useful here too?

```suggestion
a|Configures the general policy for when checkpoints should occur. The default policy is the 'periodic' checkpoint policy, as specified by the `<<config_db.checkpoint.interval.tx,db.checkpoint.interval.tx>>` and `<<config_db.checkpoint.interval.time,db.checkpoint.interval.time>>` settings. The Neo4j Enterprise Edition provides two alternative policies: The first is the 'continuous' checkpoint policy, which will ignore those settings and run the checkpoint  process all the time. The second is the 'volumetric' checkpoint policy, which makes a best-effort at checkpointing often enough so that the database doesn't get too far behind on deleting old transaction logs in accordance with the `<<config_db.tx_log.rotation.retention_policy,db.tx_log.rotation.retention_policy>>` setting.
```

Reply:
Yes, reviewing all descriptions is on my list.